modules/mapper-extras/src/test/java/org/elasticsearch/index/mapper/extras/ScaledFloatFieldMapperTests.java

```diff
@@ -432,6 +433,39 @@ protected IngestScriptSupport ingestScriptSupport() {
         throw new AssumptionViolatedException("not supported");
     }
 
+    @Override
+    protected List<NumberFieldMapperTests.OutOfRangeSpec> outOfRangeSpecs() {
+        return Collections.emptyList(); // unimplemented
+    }
+
+    @Override
+    public void testIgnoreMalformedWithObject() {} // unimplemented
```

Comment:
I've overridden the methods for now but I'm sure there's a cleaner way to write these. I've seen that some tests can be skipped in gradle, would that be cleaner, and if so, where should I look to find an example?